src/planscape/stands/management/commands/calculate_metrics.py

```diff
@@ -65,6 +65,36 @@ def add_arguments(self, parser) -> None:
 
         parser.add_argument("--force-yes", action="store_true")
 
+        parser.add_argument(
+            "--only-script",
+            action="store_true",
+            help="If true, this will only generate the output script.",
+        )
+
+    def create_load_script(self, output_folder):
+        path = Path(output_folder)
```

Comment:
Nit: this seems more like the source of csv files than an output folder, yes? Could we name this to represent what's supposed to be in the csvs?

Reply:
Oh, right, misread this. Nevermind